cypress/e2e/tests/pages/cluster-manager.spec.ts

```diff
@@ -115,7 +115,7 @@ describe('Cluster Manager', () => {
   });
 
   describe('Imported', () => {
-    const importClusterPage = new ClusterManagerImportGenericPagePo();
+    const importClusterPage = new ClusterManagerImportGenericPagePo('local');
```

Comment:
wouldit be possible to keep this empty, and the default in ClusterManagerImportPagePo be the 'blank' cluster id of `_`? it's on odd section of the ui that needs a cluster... but doesn't need a cluster.

Reply:
Thanks for your feedback @richard-cox! I've made all updates here per your change requests https://github.com/rancher/dashboard/pull/8963/commits/88631be592b76f50b9ad6a2ea182d33a9a364e71 but I wasn't sure what you meant by this request could you elaborate?

Reply:
For this round, `ClusterManagerImportGenericPagePo('local')` --> `ClusterManagerImportGenericPagePo('_')` will work

Reply:
Hmm when I make that change other areas of the test fail because `local` is hard coded in the url within some of the po files. We would need to add `clusterId` as an argument to make the urls dynamic.

For example, in `cypress/e2e/po/detail/provisioning.cattle.io.cluster/cluster-detail.po.ts` 

Currently  the `createPath` func is this:
```
private static createPath(clusterName: string, tab?: string) {
    return `/c/local/manager/provisioning.cattle.io.cluster/fleet-default/${ clusterName }`;
  }
```

It would need to update to this:
```
  private static createPath(clusterName: string, clusterId: string, tab?: string) {
    return `/c/${ clusterId }/manager/provisioning.cattle.io.cluster/fleet-default/${ clusterName }`;
  }
```